pkg/auth/serviceaccount.go

```diff
@@ -94,3 +97,30 @@ func EnsureOIDCServiceAccountExistsForResource(ctx context.Context, serviceAccou
 
 	return nil
 }
+
+type OIDCStatusMarker interface {
```

Comment:
```suggestion
type OIDCIdentityStatusMarker interface {
```
Maybe this is clearer